src/returner.py

```diff
@@ -113,7 +113,7 @@ def VANET_listening_thread():
             pkt = vanet.recv_packets()
 
             if pkt:
-                print(myUName.decode('Message',pkt[0]))
+                #print(myUName.decode('Message',pkt[0]))
```

Comment:
Is this just commented out since we are decoding elsewhere? Also- are the broadcast/returner even being used where this change is needed or is this leftover from testing?

Reply:
the broadcaster/returner are useful for testing, but not needed for anything else. I say we leave them in